.github/workflows/build_test.yml

```diff
@@ -34,41 +34,15 @@ jobs:
       fail-fast: false
 
     container:
-      image: ghcr.io/pyne/pyne_ubuntu_20.04_py3${{ matrix.hdf5 }}/${{ matrix.stage }}:stable
+      image: ghcr.io/pyne/pyne_ubuntu_22.04_py3${{ matrix.hdf5 }}/${{ matrix.stage }}:stable
```

Comment:
There isn't a good way to achieve consistency because env variables can't be used in `image:`. Furthermore, there's no way to ensure consistency across `docker_publish.yml` and `build_test.yml` because `image:` can't be in the composite action. We do have the option to remove the ubuntu version from the image name since that's something that we specify in the env variable `DOCKER_IMAGE_BASENAME`.

Reply:
Yes - I was wondering we can just do without the ubuntu version given the streamlining we are doing.  We can probably remove the `py3` as well since we are always using Python 3 now.  In short, maybe all identifiers should be machine-generated rather than hard coded.